.gitignore

```diff
@@ -3,4 +3,4 @@
 recipes/build-info/files/.build_info
 **/debian-packages.list
 rugpi-bakery.toml
-*.log
\ No newline at end of file
+*.log
```

Comment:
Generally adding unexpected newlines should be avoided if possible as it just detracts from the other changes...can you revert this whole file?

Reply:
I pushed the fix.